src/olympia/addons/tasks.py

```diff
@@ -386,3 +388,28 @@ def update_latest_version(ids, **kw):
 
     for addon in addons:
         addon.update_version()
+
+
+@task
+@write
+def populate_e10s_feature_compatibility(ids, **kwargs):
+    log.info(
+        '[%s@%s] Populating e10s feature compatibility '
+        'on addons starting w/ id: %s...' % (
+            len(ids), populate_e10s_feature_compatibility.rate_limit, ids[0]))
+
+    addons = Addon.unfiltered.filter(pk__in=ids)
+
+    for addon in addons:
+        file_ = addon.get_latest_file()
+        if file_:
+            try:
+                parsed = parse_xpi(file_.file_path, addon=addon)
+            except ValidationError:
+                log.warn('Could not parse XPI for addon %s, file %s' % (
```

Comment:
I'm not too worried because it's a one shot - we'll only need the log if, after we've launched the command, we find an add-on that wasn't updated.